analysis/reports/templates/report_version2.html.j2

```diff
@@ -35,17 +63,17 @@
             <thead>
             <tr>
                 <th>Dimension: {{ dimension }}</th>
-                <th>Significance of Dimension Value</th>
+                <th>Change Distance</th>
                 <th>Contribution to Overall Change</th>
-                <th>Change in Contribution to Total</th>
+                <th>Change in Contribution</th>
```

Comment:
Should this reference `Proportion`?  There might be some other locations as well.

Reply:
Ignore this comment, I see you've addressed it in https://github.com/mozilla/overwatch-mvp/pull/14